packages/feedback/src/util/sendFeedbackRequest.ts

```diff
@@ -34,86 +34,104 @@ export async function sendFeedbackRequest({
     type: 'feedback',
   };
 
-  const feedbackEvent = await prepareFeedbackEvent({
-    scope,
-    client,
-    event: baseEvent,
-  });
+  return new Promise((resolve, reject) => {
+    hub.withScope(async scope => {
+      // No use for breadcrumbs in feedback
+      scope.clearBreadcrumbs();
```

Comment:
just a note, nothing to do here, but this _may_ change a bit in v8. We're thinking to rework how/where we keep breadcrumbs, in the browser we'll probably keep them globally. At this point we'll have to change this logic here. Feel free to leave this for later, but if there is an easy way to make this forwards-compatible, we may as well do it already now 👍

Reply:
Good to know, would it be better to delete breadcrumbs from the event rather than the scope?